about.html

```diff
@@ -10,7 +10,7 @@ <h3 id="what">What is the SRCF?</h3>
 
 <p>The SRCF is the University of Cambridge's Student-Run Computing
 Facility. We provide web hosting, shell accounts and various other
-computing services to members of the University and groups such as
+computing services to members of the University (including alumni) and groups such as
```

Comment:
I'm not sure about the language here -- are alumni technically still members of the university? If not, maybe this should be "...computing services to members of the University, alumni, and groups..."